js/lykke2.js

```diff
@@ -0,0 +1,1174 @@
+            'lastTradeTimestamp': lastTradeTimestamp,
+            'symbol': symbol,
+            'type': type,
+            'timeInForce': undefined,
+            'postOnly': undefined,
+            'side': side,
+            'price': price,
+            'stopPrice': undefined,
+            'amount': amount,
+            'cost': cost,
+            'average': undefined,
+            'filled': filled,
+            'remaining': remaining,
+            'status': status,
+            'fee': undefined,
+            'trades': undefined,
+        }, market);
+    }
+
+    async createOrder (symbol, type, side, amount, price = undefined, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const query = {
+            'assetPairId': market['id'],
+            'side': this.capitalize (side),
+            'volume': amount,
+        };
+        if (type === 'limit') {
+            query['price'] = price;
+        }
+        const method = 'privatePostOrders' + this.capitalize (type);
+        const result = await this[method] (this.extend (query, params));
+        //
+        // market
+        //
+        //         {
+        //             "payload":{
+        //                 "orderId":"2b98ec26-8410-49b6-9f37-1fb2150e2299",
+        //                 "price":280.699
+        //             },
+        //             "error":null
+        //         }
+        //
+        // limit
+        //
+        //         {
+        //             "payload":{
+        //                 "orderId":"27be8802-30be-40ca-bf40-ec886b309c5b"
+        //             },
+        //             "error":null
+        //         }
+        //
+        const payload = this.safeValue (result, 'payload');
+        const id = this.safeString (payload, 'orderId');
+        if (type === 'market') {
+            price = this.safeNumber (payload, 'price');
+        }
+        return {
+            'id': id,
+            'info': result,
+            'clientOrderId': undefined,
+            'timestamp': undefined,
+            'datetime': undefined,
+            'lastTradeTimestamp': undefined,
+            'symbol': symbol,
+            'type': type,
+            'side': side,
+            'price': price,
+            'amount': amount,
+            'cost': undefined,
+            'average': undefined,
+            'filled': undefined,
+            'remaining': undefined,
+            'status': undefined,
+            'fee': undefined,
+            'trades': undefined,
+        };
+    }
+
+    async cancelOrder (id, symbol = undefined, params = {}) {
+        const request = {
+            'OrderId': id,
+        };
+        //
+        //     {
+        //         "payload":null,
+        //         "error":null
+        //     }
+        //
+        return await this.privateDeleteOrdersOrderId (this.extend (request, params));
+    }
+
+    async cancelAllOrders (side, symbol = undefined, params = {}) {
```

Comment:
We should make the signature of this method compatible with the other exchanges, cannot add the `side` argument to it.